drivers/tee/optee/Kconfig

```diff
@@ -15,3 +15,34 @@ config OPTEE_SHM_NUM_PRIV_PAGES
 	help
 	  This sets the number of private shared memory pages to be
 	  used by OP-TEE TEE driver.
+
+config OPTEE_BENCHMARK
```

Comment:
Do we need to add comment to indicate that benchmark is not supported in x86?

Reply:
Will comment it for x86.